x-pack/plugins/streams_app/public/components/stream_detail_management/index.tsx

```diff
@@ -90,3 +111,79 @@ export function StreamDetailManagement({
     </EuiFlexGroup>
   );
 }
+
+function assetToLink(asset: { type: string; id: string }) {
```

Comment:
These links will be broken in environments that include a `basePath`, we should use the utils from core to ensure that's prepended.

Reply:
I added the `basePath.prepend()` call to the actual `href` on the link.